service/src/main/java/cz/muni/fi/pa165/facade/TimelineFacadeImpl.java

```diff
@@ -100,6 +101,11 @@ public void removeEvent(Long timelineId, Long eventId) {
         return beanMappingService.mapTo(timelineService.findAllBetweenDates(from, to), TimelineDTO.class);
     }
 
+    @Override
+    public List<EventDTO> findEventsOfTimeline(Long timelineId) {
```

Comment:
Again I feel like this method is useless. Can't you get to events somehow by getting the instance of timeline? Ask Zloutek how he did it if you are not sure...